woocommerce/woocommerce-gateway-stripe
# UPE: migrate the upe javascript to use the wp-scripts pipeline (webpack)

# Changes proposed in this Pull Request:

Issue: 
Fixes #

This PR migrate the UPE javascript for classic checkout from normal js asset + minified, to use the wp-scripts pipeline (webpack)

# Testing instructions
- Enable UPE feature flag: 
```sql
INSERT INTO `wp_options` (`option_id`, `option_name`, `option_value`, `autoload`) VALUES (NULL, '_wcstripe_feature_upe', '1', 'no') ON DUPLICATE KEY UPDATE `option_value`=VALUES(`option_value`);
```
- Run `npm install`
- Run `npm run start:webpack`
- Check that the folder `/build` contains the following files: `style-upe_classic.css`, `upe_classic.js` and `upe_classic.asset.php`

- Go to WooCommerce > Settings > Payments > Stripe settings
- Check the "Enable new checkout experience" option and hit save
- Scroll to the bottom and there should now be a table of UPE payment methods, make sure Card is turned on.

- Go to `http://localhost:8082/checkout/`, and check that the Credit Card form loads correctly.
